### ISSUE TITLE
CDKTF plan/apply not using Structured Run output in Terraform Cloud

### ISSUE THREAD

**Issue by sabinayakc** [hashicorp/terraform-cdk#2597]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

language: typescript
cdktf-cli: 0.15.2
node: v16.17.0
cdktf: null
constructs: 10.1.162
jsii: null
terraform: 1.3.7
arch: x64
os: linux 5.10.102.1-microsoft-standard-WSL2

### Affected Resource(s)

`cdktf plan`
`cdtkf apply`

### My Stack Code
```
  new CloudBackend(eksStack, {
    hostname: "app.terraform.io",
    organization: "XXXXX",
    workspaces: new NamedCloudWorkspace("eks-cluster-1"),
  });
```

### Expected Behavior

The plan should use Structured Run output instead of Console UI. 
This is when I use v0.14.3,
![image](https://user-images.githubusercontent.com/13154166/216733919-0e9f59f7-6411-40da-8971-2c5a291bd44f.png)

### Actual Behavior

The plan is using Console UI output instead.

![image](https://user-images.githubusercontent.com/13154166/216732799-249bd9bb-a2f9-4cf5-b9d3-3caa12abff35.png)

### Steps to Reproduce

- Create a workspace in TFC
- Select Structure Run Output from General Settings
- 
![image](https://user-images.githubusercontent.com/13154166/216733527-bb9e71bd-3d35-4291-b884-5fc18d0f13fe.png)

- Run `cdktf plan` or `apply`

### Important Factoids

- I think it has to do with using *API driven workflow (v0.14.3)* vs *CLI driven workflow (> v0.15)*

**Comment by ansgarm** [comment]
Hi @sabinayakc 👋

Thank you for reporting this! This [PR](https://github.com/hashicorp/terraform/pull/32504) to the Terraform CLI will fix this, it seems to be targeted for Terraform 1.4 👍

**Comment by xiehan** [comment]
This should be resolved now that Terraform 1.4 is out.
